components/ui/card.tsx

```diff
@@ -0,0 +1,146 @@
+import * as React from 'react'
+import { cn } from '@/lib/utils'
+import { CardI } from '@/lib/interface/controls/card'
+import Image from 'next/image'
+
+const Card = React.forwardRef<HTMLDivElement, CardI>(
+  (
+    {
+      className,
+      title,
+      children,
+      variant = 'default',
+      primaryColor = 'default',
+      imgUrl,
+      members,
+      subject,
+      desc,
+      Icon,
+      ...props
+    },
+    ref
+  ) => (
+    <>
+      {variant === 'default' && (
+        <div
+          ref={ref}
+          className={cn(
+            'rounded-lg border border-slate-200 bg-white  min-w-[280px] max-w-[300px] h-[160px] text-slate-950 shadow-sm dark:border-slate-800 dark:bg-slate-950 dark:text-slate-50',
+            className
+          )}
+          {...props}
+        >
+          <CardHeader
+            className={`${primaryColor == 'default' ? 'bg-[#3A86FF]' : 'bg-[#EECB51]'} font-medium rounded-t-lg flex-row   flex items-center gap-2`}
```

Comment:
Use `cn` here to conditionally apply the color classes, and **always** use a strict equality check (**===** instead of the plain **==**).

For example:

```
className={cn('font-medium ...', {
  'bg-[#3A86FF]': primaryColor === 'default',
  'bg-[#EECB51]': primaryColor !== 'default
})}
```

There are also three spaces between `flex-row` and `flex`. Also, `flex-row` is not a valid tailwind class; `flex` does what you want.